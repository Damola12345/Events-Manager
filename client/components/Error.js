import React from 'react';
import { Link } from 'react-router-dom';

const Footer = () => {
    return (
      <div>
        <h1>ERROR!!!</h1>
      </div>
    );
};

export default Footer;