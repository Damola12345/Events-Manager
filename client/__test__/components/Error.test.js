import React from 'react';
import { shallow } from 'enzyme';
import Error from '../../components/Error';

describe('Error', () => {
  const wrapper = shallow(<Error />);
  it('successfully rendered', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

