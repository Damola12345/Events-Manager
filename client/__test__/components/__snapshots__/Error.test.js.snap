// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error successfully rendered 1`] = `
<div>
  <h1>
    404
  </h1>
  <h3>
    Not Found!
  </h3>
</div>
`;
